From 6658bdc1efac6ba15bee46587215cf9464de7dc4 Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@Freescale.com>
Date: Sat, 18 Jan 2014 04:07:28 -0600
Subject: [PATCH 076/466] fsl_qman: Fix typo in exporting
 qman_get_affine_portal()

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
Change-Id: I8b334cc6f170ff882633efe032e01d45d6d7c173
Reviewed-on: http://git.am.freescale.net:8181/8105
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Haiying Wang <Haiying.Wang@freescale.com>
Reviewed-by: Emilian Medve <Emilian.Medve@freescale.com>
[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/staging/fsl_qbman/qman_high.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl_qbman/qman_high.c b/drivers/staging/fsl_qbman/qman_high.c
index dfc1bc9..0e0e23a 100644
--- a/drivers/staging/fsl_qbman/qman_high.c
+++ b/drivers/staging/fsl_qbman/qman_high.c
@@ -1119,7 +1119,7 @@ void *qman_get_affine_portal(int cpu)
 {
 	return affine_portals[cpu];
 }
-EXPORT_SYMBOL(qman_affine_portal);
+EXPORT_SYMBOL(qman_get_affine_portal);
 
 int qman_p_poll_dqrr(struct qman_portal *p, unsigned int limit)
 {
-- 
1.7.10.4

